cmd/log_drains.go

```diff
@@ -27,7 +36,19 @@ var (
 				return
 			}
 
-			err := log_drains.List(currentApp)
+			var addonID string
+			if c.GlobalString("addon") != "<addon_id>" {
+				addonID = c.GlobalString("addon")
+			} else if c.String("addon") != "<addon_id>" {
+				addonID = c.String("addon")
+			}
+
+			var err error
+			if addonID != "" || c.String("with-addon") == "true" {
```

Comment:
`with-addon` being a `BoolFlag`, you should get its value with `Bool`:

```suggestion
			if addonID != "" || c.Bool("with-addon") {
```